samples/modules/lvgl/demos/README.rst

```diff
@@ -0,0 +1,48 @@
+.. zephyr:code-sample:: lvgl-demos
+   :name: LVGL demos
+   :relevant-api: display_interface
+
+   Run LVGL built-in demos.
+
+Overview
+********
+
+A sample showcasing upstream LVGL demos.
+
+* Music: The music player demo shows what kind of modern, smartphone-like user interfaces can be created on LVGL.
+* Benchmark: The benchmark demo tests the performance in various cases. For example rectangle, border, shadow, text, image blending, image transformation, blending modes, etc.
+* Stress: A stress test for LVGL. It contains a lot of object creation, deletion, animations, styles usage, and so on. It can be used if there is any memory corruption during heavy usage or any memory leaks.
```

Comment:
Just looked at the [rendered doc page](https://builds.zephyrproject.io/zephyr/pr/62904/docs/samples/modules/lvgl/demos/README.html#lvgl-demos) and realized a definition list would probably be more appropriate here:

```suggestion
* Music
      The music player demo shows what kind of modern, smartphone-like user interfaces can be created on LVGL.
* Benchmark
      The benchmark demo tests the performance in various cases. For example rectangle, border, shadow, text, image blending, image transformation, blending modes, etc.
* Stress
      A stress test for LVGL. It contains a lot of object creation, deletion, animations, styles usage, and so on. It can be used if there is any memory corruption during heavy usage or any memory leaks.
```